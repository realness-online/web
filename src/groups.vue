<template lang="html">
  <section id="groups">
    <header>
      <h1>Groups</h1>
    </header>
  </section>
</template>

<script>
  export default {
    data() {
      return {}
    }
  }
</script>

<style lang="stylus">
  @require 'style/variables'
  section#groups
    padding: 0 base-line
</style>
