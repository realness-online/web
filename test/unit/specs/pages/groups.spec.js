import Vue from 'vue'
import { shallow } from 'vue-test-utils'
import groups from '@/pages/groups'

describe('@/pages/groups.vue', () => {
  it('renders group layout', () => {
    let wrapper = shallow(groups)
    expect(wrapper.element).toMatchSnapshot()
  })
})
