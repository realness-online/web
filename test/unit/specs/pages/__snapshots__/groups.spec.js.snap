// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@/pages/groups.vue renders group layout 1`] = `
<section
  id="groups"
>
  <header>
    <h1>
      Groups
    </h1>
  </header>
</section>
`;
